lib/src/firestore.dart

```diff
@@ -76,6 +76,12 @@ class Firestore {
     return new CollectionReference(nativeInstance.collection(path), this);
   }
 
+  /// Gets a [DocumentQuery] for the specified Firestore path.
```

Comment:
This doc seems inaccurate. Maybe use the one from the JS binding?

Reply:
Doc comment updated.